app/models/qualification.rb

```diff
@@ -0,0 +1,11 @@
+class Qualification < ApplicationRecord
+  belongs_to :job_application
+
+  enum category: { gcse: 0, as_level: 1, a_level: 2, other_secondary: 3, undergraduate: 4, postgraduate: 5, other: 6 }
+
+  def name
+    return read_attribute(:name) if read_attribute(:name).present? || other? || other_secondary?
```

Comment:
I'm a bit confused by what's going on here, is there something I'm missing?

Reply:
I created this `name` method a bit early, will use it next PR, but the idea is that

- Some types of qualification form don't have a 'name' field (GCSE etc) and others do have a 'name' (non-standard qualifications).
- We want to display the name as part of the DetailsComponent/ReviewComponent.
- If the name column is nil in the database (e.g. it's a GCSE), fall back to using the category instead when 'name' is called.
- Now any qualification can have 'name' called and reliably return something useful to display.